crates/bevy_pbr/src/lib.rs

```diff
@@ -80,14 +82,16 @@ impl Plugin for PbrPlugin {
                 // add as an exclusive system
                 add_clusters
                     .exclusive_system()
-                    .label(SimulationLightSystems::AddClusters),
+                    .label(SimulationLightSystems::AddClusters)
+                    .after(ModifiesWindows),
```

Comment:
I think now that change_windows is not exclusive, this will cause a warning to be logged that this system tried to be scheduled after ModifiesWindows and that system doesn’t exist in the exclusive part of PostUpdate.

Reply:
Removed it